reference-architecture/aws-ansible/README.md

```diff
@@ -15,10 +15,12 @@ The code in this repository handles all of the AWS specific components except fo
 ```
 $ *subscription-manager repos --enable rhel-7-server-optional-rpms*
 $ *subscription-manager repos --enable rhel-7-server-ose-3.2-rpms*
+$ *subscription-manager repos --enable rhel-7-server-ose-3.3-rpms*
 $ *rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm*
 $ *yum -y install atomic-openshift-utils \ *
   *               python2-boto \ *
   *               git \ *
+  *               ansible-2.2.0-0.5.prerelease.el7.noarch \ *
```

Comment:
which version of ansible is available for centos/fedora users? i managed to find ony 2.1 now... and workaround was to use full path in the could-infra task. maybe if for now its not available worth moving this to separate task for RH based OS only, and cover workaround in readme?
workaround:
````
- name: Create Greenfield Infrastructure
  cloudformation:
    stack_name: "OpenShift-Infra"
    state: "present"
    region: "{{ region }}"
    template: "/home/mangis/workspace/openshift-ansible-contrib/reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/greenfield.json"
````

 my ansible availability: 
````
[mangis@mj .ssh]$ yum info ansible.noarch
Redirecting to '/usr/bin/dnf info ansible.noarch' (see 'man yum2dnf')

Last metadata expiration check: 2 days, 4:00:33 ago on Sat Oct  1 18:48:43 2016.
Installed Packages
Name        : ansible
Arch        : noarch
Epoch       : 0
Version     : 2.1.1.0
Release     : 1.fc24
Size        : 15 M
Repo        : @System
From repo   : @commandline
Summary     : SSH-based configuration management, deployment, and task execution system
URL         : http://ansible.com
License     : GPLv3+
Description : 
            : Ansible is a radically simple model-driven configuration management,
            : multi-node deployment, and remote task execution system. Ansible works
            : over SSH and does not require any software or daemons to be installed
            : on remote nodes. Extension modules can be written in any language and
            : are transferred to managed machines automatically.

[mangis@mj .ssh]$ cat /etc/fedora-release 
Fedora release 24 (Twenty Four)
[mangis@mj .ssh]$ 
````